scripts/inventory/vagrant.py

```diff
@@ -57,26 +54,27 @@
 #
 # helper functions
 #
-
-
 # get all the ssh configs for all boxes in an array of dictionaries.
-def get_ssh_config():
-    return dict((k, get_a_ssh_config(k)) for k in list_running_boxes())
-
-
 # list all the running boxes
-def list_running_boxes():
 
-    output = to_text(subprocess.check_output(["vagrant", "status"]), errors='surrogate_or_strict').split('\n')
+output = to_text(subprocess.check_output(["vagrant", "global-status", "--prune"]), errors='surrogate_or_strict').split('\n')
```

Comment:
Is there a reason why you make this code global, instead of keeping it in a function (which is a lot cleaner)?